### ISSUE TITLE
Time should be tied to Home assistant time zone

### ISSUE THREAD

**Issue by alfwro13** [lozzd/octopus-energy-rates-card#75]
I was recently traveling and I have noticed that when I was in a different time zone that was +1hour in comparison to my home location the card was showing only slots my current location. The only work around was to change time on my mobile phone to the home assistant time to see it correctly. 
I think this card should be tied somehow to the home assistant time possibly using https://www.home-assistant.io/integrations/time_date/

**Comment by lozzd** [comment]
Agree! I travel a lot for work and get so confused when it's in the wrong timezone. Will see if I can figure this out (or hopefully someone else beats me!)

**Comment by swests** [comment]
I’d like to +1 this.  I’ve got the HA companion app set up to default to server TZ, but this integration defaults to browser TZ.

**Comment by swests** [comment]
Thinking about it, could your JS leverage sensor.date_time from the Date & Time integration?

**Comment by elyobelyob** [comment]
Just wondering why this is still an issue. I've to got abroad and it still doesn't use the HA timezone. Would really appreciate this fixed! Yes, may take a look myself.
